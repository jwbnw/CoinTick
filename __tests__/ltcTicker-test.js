import React from "react";
import LtcTicker from "./../components/ltcTicker";

import renderer from "react-test-renderer";

test("Renders Correctly", () => {
  const tree = renderer.create(<LtcTicker />).toJSON();
  expect(tree).toMatchSnapshot();
});
