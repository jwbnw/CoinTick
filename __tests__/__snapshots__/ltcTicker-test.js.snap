// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
      "padding": 10,
      "textAlign": "center",
    }
  }
>
  <Image
    source={Object {}}
    style={
      Object {
        "alignSelf": "center",
      }
    }
  />
</View>
`;
